testing/sdk_utils.py

```diff
@@ -165,6 +166,22 @@ def get_metadata() -> requests.Response:
                                    retry=False)
 
 
+def get_cluster_zones():
+    rc, stdout, _ = sdk_cmd.run_cli("node --json")
+
+    if rc != 0:
+        log.warning("return code for command 'node --json' is non-zero: %s", rc)
+        return None
+
+    r = json.loads(stdout)
+    ip_zn = {}
```

Comment:
Attempting to improve variable name to be more descriptive:
```suggestion
    zones_by_ip = {}
```
or
```suggestion
    ips_to_zone = {}
```